core/db/migrate/20150609093816_increase_scale_on_pre_tax_amounts.rb

```diff
@@ -0,0 +1,16 @@
+class IncreaseScaleOnPreTaxAmounts < ActiveRecord::Migration
+  def change
+    # set pre_tax_amount on shipments to discounted_amount - included_tax_total
+    # so that the null: false option on the shipment pre_tax_amount doesn't generate
+    # errors.
+    #
```

Comment:
If you look two lines lower in this file the table name is also hardcoded. Table names should be hardcoded in migrations, we don't want migrations to change behaviour.

SQL is always preferable to AR in migrations for the same reason.